scripts/pathseq/wdl/pathseq_pipeline_template.json

```diff
@@ -1,19 +1,41 @@
 {
-  "PathSeqPipelineWorkflow.sample_name": "sample",
-  "PathSeqPipelineWorkflow.input_bam": "gs://my-bucket/sample.bam",
+  "##_COMMENT1": "Inputs",
+  "PathSeqPipelineWorkflow.sample_name": "NA12878_24RG_med.hg380.7chicken0.3",
+  "PathSeqPipelineWorkflow.input_bam": "gs://gatk-best-practices/pathseq/contaminated-bam/NA12878_24RG_med.hg380.7chicken0.3.bam",
 
-  "PathSeqPipelineWorkflow.is_host_aligned": false,
-  "PathSeqPipelineWorkflow.min_clipped_read_length": 60,
-  "PathSeqPipelineWorkflow.filter_duplicates": true,
+  "##_COMMENT1": "Reference and Resources",
+  "PathSeqPipelineWorkflow.kmer_file": "gs://gatk-best-practices/pathseq/resources/pathseq_host.bfi",
+  "PathSeqPipelineWorkflow.microbe_bwa_image": "gs://gatk-best-practices/pathseq/resources/meats.fa.img",
+  "PathSeqPipelineWorkflow.microbe_fasta": "gs://gatk-best-practices/pathseq/resources/meats.fa",
+  "PathSeqPipelineWorkflow.microbe_fasta_dict": "gs://gatk-best-practices/pathseq/resources/meats.dict",
+  "PathSeqPipelineWorkflow.filter_bwa_image": "gs://gatk-best-practices/pathseq/resources/pathseq_host.fa.img",
+  "PathSeqPipelineWorkflow.taxonomy_file": "gs://gatk-best-practices/pathseq/resources/meats.min2k.db",
+  "PathSeqPipelineWorkflow.is_host_aligned": "false",
+
+  "##_COMMENT1": "Docker",
+  "PathSeqPipelineWorkflow.gatk_docker_override": "broadinstitute/gatk:4.0.11.0",
```

Comment:
@bshifaw  -- @mwalker174 has some changes in PR for PathSeq, so maybe hold off on this PR until those go in so we can make sure everything is up to date.